cloudwatchevents/guarddutybenchmark/template_v2.yaml

```diff
@@ -85,7 +86,7 @@ Parameters:
       - us1
       - in
       - fed
-    Description: "Enter au, ca, de, eu, jp, us2, or us1"
+    Description: "Enter in, fed, au, ca, de, eu, jp, us2, or us1"
```

Comment:
I haven't tested it on sumo's fed deployment. In fed there are some features/APIs not available like C2C so I don't know whether it works there or not.

Reply:
fed was already part of list. I just updated the description.

Reply:
hmm the list is not a dropdown it's a validation I guess since the description is visible to customers so that's why they didn't tried entering fed but ok lets see if anyone complaints.